examples/api/test/widgets/scrollbar/raw_scrollbar.0_test.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/scrollbar/raw_scrollbar.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('There are 2 scrollbars', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('There are two scrollbars', (WidgetTester tester) async {
```
Not sure, if there a rule for that, but it is probably better to keep real number for things such as offset, size, etc